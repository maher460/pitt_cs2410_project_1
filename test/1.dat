ld R1, 0(R0)
ld R2, 4(R0)
ld R3, 8(R0)
ld R4, 12(R0)
ld R5, 16(R0)
ld R6, 20(R0)
ld R7, 24(R0)
ld R8, 28(R0)
and R9, R1, R2
andi R10, R2, 2
or R11, R3, R4
ori R12, R5, 2
slt R13, R11, R12
or R14, R13, R0
slti R14, R14, 7
add R15, R6, R7
sub R16, R7, R8
mul R17, R7, R8
sd R12, 100(R0)
sd R13, 104(R0)
sd R14, 108(R0)
DATA
Mem(0) = 1.0
Mem(4) = 2.0
Mem(8) = 3.0
Mem(12) = 4.0
Mem(16) = 5.0
Mem(20) = 6.0
Mem(24) = 7.0
Mem(28) = 8.0
Mem(100) = 0.0
Mem(104) = 0.0
Mem(108) = 0.0