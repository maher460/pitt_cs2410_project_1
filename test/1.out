BASIC int

----------
Data Cache
----------
Mem(0) = 1.0
Mem(4) = 2.0
Mem(8) = 3.0
Mem(12) = 4.0
Mem(16) = 5.0
Mem(20) = 6.0
Mem(24) = 7.0
Mem(28) = 8.0
Mem(100) = 7.0
Mem(104) = 0.0
Mem(108) = 1.0
-----------------
Integer Registers
-----------------
Int_Reg[0]=0
Int_Reg[1]=1
Int_Reg[2]=2
Int_Reg[3]=3
Int_Reg[4]=4
Int_Reg[5]=5
Int_Reg[6]=6
Int_Reg[7]=7
Int_Reg[8]=8
Int_Reg[9]=0
Int_Reg[10]=2
Int_Reg[11]=7
Int_Reg[12]=7
Int_Reg[13]=0
Int_Reg[14]=1
Int_Reg[15]=13
Int_Reg[16]=-1
Int_Reg[17]=56
Int_Reg[18]=0
Int_Reg[19]=0
Int_Reg[20]=0
Int_Reg[21]=0
Int_Reg[22]=0
Int_Reg[23]=0
Int_Reg[24]=0
Int_Reg[25]=0
Int_Reg[26]=0
Int_Reg[27]=0
Int_Reg[28]=0
Int_Reg[29]=0
Int_Reg[30]=0
Int_Reg[31]=0
------------------------
Floating Point Registers
------------------------
FP_Reg[0]=0.0
FP_Reg[1]=0.0
FP_Reg[2]=0.0
FP_Reg[3]=0.0
FP_Reg[4]=0.0
FP_Reg[5]=0.0
FP_Reg[6]=0.0
FP_Reg[7]=0.0
FP_Reg[8]=0.0
FP_Reg[9]=0.0
FP_Reg[10]=0.0
FP_Reg[11]=0.0
FP_Reg[12]=0.0
FP_Reg[13]=0.0
FP_Reg[14]=0.0
FP_Reg[15]=0.0
FP_Reg[16]=0.0
FP_Reg[17]=0.0
FP_Reg[18]=0.0
FP_Reg[19]=0.0
FP_Reg[20]=0.0
FP_Reg[21]=0.0
FP_Reg[22]=0.0
FP_Reg[23]=0.0
FP_Reg[24]=0.0
FP_Reg[25]=0.0
FP_Reg[26]=0.0
FP_Reg[27]=0.0
FP_Reg[28]=0.0
FP_Reg[29]=0.0
FP_Reg[30]=0.0
FP_Reg[31]=0.0

commit instructions:21